value.go

```diff
@@ -57,3 +57,16 @@ func (v *Value) IsAnonymousType() bool {
 func (v *Value) IsAmbiguousType() bool {
 	return v.IsAmbiguousDynamicType || v.IsAmbiguousStaticType
 }
+
+// CanNil reports if the Value can be nil as a zero value. When this method
```

Comment:
```suggestion
// CanNil reports if v.Value can be nil. When this method
```

I'm not sure if the zero-value bit is relevant, can you clarify?

Reply:
Yeah, it sounds like it is not relevant to the problem this change is trying to solve.  Committing the comment revision...